src/cmakeFindModules/FindOpenImageIO.cmake

```diff
@@ -0,0 +1,18 @@
+#find headers
```

Comment:
I know, not enthusiastic either.
The thing is that OIIO can write exif while openCV can't. It's difficult to find a library for exif I/O with full support and the proper license.